types/mongodb/index.d.ts

```diff
@@ -2171,7 +2172,7 @@ export interface FindOperatorsUnordered {
 /** http://mongodb.github.io/node-mongodb-native/3.6/api/Collection.html#findOne */
 export interface FindOneOptions<T> {
     limit?: number;
-    sort?: Array<[string, number]> | SortOptionObject<T>;
+    sort?: Array<[string, SortValues]> | SortOptionObject<T>;
```

Comment:
Could you make sure the same fix is applied everywhere `sort` can be specified? I believe there are a few places across the document where this fix could be made.

Reply:
Good idea. I found a couple of other places and changed those and pushed.